Sources/UInt128.swift

```diff
@@ -429,19 +429,15 @@ extension UInt128 : BinaryInteger {
     // MARK: Type Methods
 
     public static func /(lhs: UInt128, rhs: UInt128) -> UInt128 {
-        let result = lhs.dividedReportingOverflow(by: rhs)
-
-        return result.partialValue
+        return lhs.dividedReportingOverflow(by: rhs).partialValue
```

Comment:
I would prefer to not have the train of calls. It’s mostly for reasons matching my own escetic, but I can’t see any advantage to doing it differently then what I already have. Ditto for the next 3 places you did this.